lisa/features/infiniband.py

```diff
@@ -26,3 +50,58 @@ def is_over_sriov(self) -> bool:
     # example SKU: Standard_H16mr
     def is_over_nd(self) -> bool:
         raise NotImplementedError
+
+    def get_ib_interfaces(self) -> List[Tuple[str, str, str]]:
+        """Gets the list of Infiniband devices
+        excluding any ethernet devices
+        and get their cooresponding network interface
+        Returns list of tuples in the form (ib_device_name, nic_name, ip_addr)
+        Example ("mlx5_ib0", "ib0", "172.16.1.23")"""
+        ib_devices = []
+        device_info = self._get_ib_device_info()
+        print(f"Device Info: {device_info}")
```

Comment:
don't use print any time, use `log.debug` always.

Reply:
I forgot to remove this after debugging an issue. It has been removed.